tests/clims/api/serializers/models/test_extensible_property.py

```diff
@@ -0,0 +1,51 @@
+
+from __future__ import absolute_import
+
+from sentry.testutils import TestCase
+
+from clims.api.serializers.models.extensible_property import ExtensiblePropertySerializer
+from clims.models.extensible import ExtensibleProperty, ExtensiblePropertyType, ExtensibleType
+from clims.models.plugin_registration import PluginRegistration
+
+from sentry.models.organization import Organization
+
+
+class ExtensiblePropertyTypeSerializerTestCase(TestCase):
+
+    def setUp(self):
+        org, _ = Organization.objects.get_or_create(name="testorg")
```

Comment:
FYI: There is an organization available as a fixture for all tests in `self.organization`. I was using this pattern earlier but have moved to using only the fixtures.